src/sentry/static/sentry/app/components/assistant/supportDrawer.jsx

```diff
@@ -0,0 +1,133 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import createReactClass from 'create-react-class';
+import $ from 'jquery';
+import HookStore from '../../stores/hookStore';
+
+// SupportDrawer slides up when the user clicks on a "Need Help?" cue.
+const SupportDrawer = createReactClass({
+  displayName: 'SupportDrawer',
+
+  propTypes: {
+    closeHandler: PropTypes.func.isRequired,
+  },
+
+  getInitialState() {
+    return {
+      inputVal: '',
+      docResults: [],
+      helpcenterResults: [],
+    };
+  },
+
+  componentWillReceiveProps(props) {
+    this.setState({inputVal: ''});
+  },
+
+  handleSubmit(evt) {
+    evt.preventDefault();
+  },
+
+  handleChange(evt) {
+    evt.preventDefault();
+    let term = encodeURIComponent(evt.currentTarget.value);
+
+    this.setState({
+      inputVal: evt.currentTarget.value,
+    });
+
+    if (term == '') {
+      return;
+    }
+    $.get(
+      `https://rigidsearch.getsentry.net/api/search?q=${term}&page=1&section=hosted`,
+      data => {
+        this.setState({docResults: data.items});
+      }
+    );
+    $.get(
+      `https://sentry.zendesk.com/api/v2/help_center/articles/search.json?query=${term}`,
```

Comment:
This and the above request is something we'll want to move to getsentry right?

Reply:
Since this is docs + FAQ search that is available generally, we were planning to have it in sentry too

Reply:
Ah, so these endpoints are publicly available? I didn't know